gio/Gir.toml

```diff
@@ -327,14 +327,23 @@ generate_builder = true
 name = "Gio.Cancellable"
 status = "generate"
 concurrency = "send+sync"
+manual_traits = ["CancellableExtManual"]
+    [[object.function]]
+    name = "connect"
+    # Needs manual bindings for SignalHandlerId
+    manual = true
```

Comment:
`rename = "connect_cancelled"` etc

And also mark the `cancelled` signal ignored